bin/weekly_total.py

```diff
@@ -25,6 +25,13 @@ def main():
     """Code to query DB for a week of counts, sum them, and store result."""
     # NOTE: This works as expected if script is scheduled in UTC
     today = datetime.date.today()
+    weekday = today.isoweekday()
+    if weekday != 1:
+        # If not Monday, abandon script and exit
+        msg = ("Day of week is {} -- not Monday. "
+               "Weekly count script exited.").format(weekday)
+        LOGGER.warning(msg)
+        return
```

Comment:
empty `return` is returning None and stops the execution.
I would prefer 

```suggestion
        sys.exit()
```

Reply:
👍